specs/wp-signup-spec.js

```diff
@@ -49,6 +49,11 @@ const screenSize = driverManager.currentScreenSize();
 const signupInboxId = config.get( 'signupInboxId' );
 const host = dataHelper.getJetpackHost();
 const locale = driverManager.currentLocale();
+const personalPlanSlug = 'personal-bundle';
+const premiumPlanSlug = 'value_bundle';
+const businessPlanSlug = 'business-bundle';
+const dotLiveDomainSlug = 'dotlive_domain';
+const privateWhoisSlug = 'private_whois';
```

Comment:
I think it would better feet into `SecurePaymentComponent` - the same approach as we do with Plans page

Reply:
done 🐌